src/shared/lib/comparison/AnalysisStore.ts

```diff
@@ -0,0 +1,383 @@
+import { remoteData } from 'cbioportal-frontend-commons';
+import {
+    CancerStudy,
+    Gene,
+    Mutation,
+    MutationCountByPosition,
+    Sample,
+} from 'cbioportal-ts-api-client';
+import { computed, makeObservable, observable } from 'mobx';
+import _ from 'lodash';
+import { compileMutations } from '../../../pages/resultsView/ResultsViewPageStoreUtils';
```

Comment:
Ideally we shouldn't be referencing to `pages/resultsView` here. Might be better to move the function to `AnalysisStoreUtils`